packages/plugin-ext/src/main/browser/view/plugin-view-registry.ts

```diff
@@ -240,17 +252,7 @@ export class PluginViewRegistry implements FrontendApplicationContribution {
             id: toggleCommandId,
             label: 'Toggle ' + options.label + ' View'
         }, {
-            execute: async () => {
-                let widget = await this.getPluginViewContainer(id);
-                if (widget) {
-                    widget.dispose();
-                } else {
-                    widget = await this.openViewContainer(id);
-                    if (widget) {
-                        this.shell.activateWidget(widget.id);
-                    }
-                }
-            }
+            execute: async () => { await this.toggleViewContainer(id); }
```

Comment:
I thought I had already commented on this line but we can do the following to simplify:

```suggestion
            execute: () => this.toggleViewContainer(id);
```

Reply:
Right, I thought of it but saw [this](https://github.com/eclipse-theia/theia/blob/625c021715f0e9d1c91015049656a7d1225a6511/packages/plugin-ext/src/main/browser/view/plugin-view-registry.ts#L317) ...
Fixed.